omnichannel-server/src/feature/message-source/MessageParser.ts

```diff
@@ -0,0 +1,62 @@
+import { MessageBirdPayload } from "./interfaces/MessageBirdPayload";
+import { MessageTwilioPayload } from "./interfaces/MessageTwilioPayload";
+import { MessagePayload } from "./interfaces/MessagePayload";
+
+export const MessageParser = ({body}) => {
+    const messageBody = <MessageBirdPayload|MessageTwilioPayload>body;
```

Comment:
Buenísimo. Esto es lo que esperaba. 
Sólo falta aplicar `prettier` para que quede así: 

```suggestion
    const messageBody = <MessageBirdPayload | MessageTwilioPayload>body;
```

También, un sintaxis común es: 

```suggestion
    const messageBody: MessageBirdPayload | MessageTwilioPayload = body;
```

Reply:
A bueno ahorita lo aplico